packages/sit-onyx/src/components/OnyxEmpty/OnyxEmpty.stories.ts

```diff
@@ -0,0 +1,61 @@
+import { OnyxLink } from "@/index";
+import { createTruncationDecorator } from "@/utils/storybook";
+import emojiSad from "@sit-onyx/icons/emoji-sad.svg?raw";
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import { h } from "vue";
+import OnyxIcon from "../OnyxIcon/OnyxIcon.vue";
+import OnyxEmpty from "./OnyxEmpty.vue";
+
```

Comment:
should there be an example with a different icon size? in Figma it is a property, I understand that that is not how it works with the slot but a Dev that inspects the component in the project might search for how it is done.
<img width="690" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/be750ee1-3154-4deb-9cf2-fe3aedaafbee">

on the other hand teh dev might see in the example that OnyxIcon is used and look into the docs of OnyxIcon ^^

it's fine either way :)

<img width="721" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/149e8c3f-ccc5-46a5-82c4-76cc5dfb3abe">
when I saw this I first thought the size was about density